src/components/ScholarshipDetailCard.jsx

```diff
@@ -227,13 +234,13 @@ export default function ScholarshipDetailCard({ scholarship, preview }) {
         <Grid container>
           {!tags
```

Comment:
`tags` could be `[]`.
```suggestion
          {!tags || !tags.length
```

Alternatively we could render no tags in a different manner. Up to you